### DISCUSSION TITLE
Community Grants Program

### DISCUSSION THREAD

**Discussion by joecartesi** [cartesi/cips#37] [Ideas]
Hi, please provide any feedback you may have for this proposal: https://github.com/cartesi/cips/issues/35 Thank you!

**Comment by packetstracer** [comment]
This looks good to us, in fact we thinks is a solid proposal because of the following.

grant programs for software development help flourish interesting projects in the ecosystem
decentralization and community involvement is a good thing to promote project ownership by holders
the council member is a proven way to manage the selection and tracking of the projects
a limited program time is also nice to test and improve any issues on the program
In brief it's a solid starting point in our opinion, but would like to point a matter about point 2 (decentralization and community driven selection).

Though we think decision making decentralization is important, we would like to raise the issue that selecting technological projects in a quite technological environment such as blockchain is, requires at least good knowledge of the technologies involved as well as software development to be able to score and prioritize projects:

that are viable and can fit into the market
that are within a fair price range
where team members are capable of delivering the project
That's why we think the council is very important to ensure the health of the program and avoid the raise of "false" proposals, community and bot driven voting, and non appropriate project selection because of not adequate
knowledge.

So we think that keeping the council and not going to a full decentralization is important. We believe that some tasks like filtering projects should always still be done by a council. Other projects include the selection of the council by the community, with some constraints involved like max time a council member can stay in the council, member rotation number and time windows, etc.

Anyway, for this first program phase this is not a problem, but just wanted to point it out for future stages.

**Comment by DurableNetwork** [comment]
An open grant program is an enormous incentive for builders, but is also a very good way of testing and improving the participation of the community. Decisions involving monetary transactions like unlocking a grant are crucial in the lifecycle of an organization. Being open about those decisions, as well as listening to the community's opinion, will bring more trust and transparency to the initiative and solidify the project as a whole. The idea of decentralizing organizations is exciting, and this is a good first step on which we can iterate as we progress in the right direction!
